apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -184,9 +185,6 @@ export class ChangeAction extends BaseRushAction {
 
   private _validateChangeFile(changedPackages: string[]): void {
     const files: string[] = this._getChangeFiles();
-    if (files.length === 0) {
```

Comment:
Why are you removing this?

Reply:
This is covered by `.validate` on the next line, and the error message there provides more information.